pkg/autoupdate/tuf/autoupdate.go

```diff
@@ -0,0 +1,231 @@
+package tuf
+
+// This new autoupdater points to our new TUF infrastructure, and will eventually supersede
+// the legacy `Updater` in pkg/autoupdate that points to Notary.
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/version"
+	client "github.com/theupdateframework/go-tuf/client"
+	filejsonstore "github.com/theupdateframework/go-tuf/client/filejsonstore"
+)
+
+//go:embed assets/tuf-dev/root.json
+var rootJson []byte
+
+const (
+	DefaultTufServer = "https://tuf-devel.kolide.com"
```

Comment:
I think @directionless named it 🙂 

I could make the local directory `tuf-devel` instead of `tuf-dev` if we think it'd be better for them to match -- I wasn't thinking too hard about it when I named the directory haha

Reply:
don't need to change anything, I was just curious